README.md

```diff
@@ -1,5 +1,8 @@
 # Dojo 2 core
 
+[![Build Status](https://travis-ci.org/bryanforbes/core.svg?branch=coverage)](https://travis-ci.org/bryanforbes/core)
```

Comment:
I have this changed locally. I'll push all of the changes discussed here once we're done.